modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -1,20 +1,20 @@
 [[clients-ubuntu]]
-= Registering {ubuntu} 20.04, 22.04 and 24.04 Clients
+= Registering {ubuntu} 22.04 and 24.04 Clients
```

Comment:
I would remove client versions from the title as well.  We don't have them for any other OS. Master only, no need for back-porting if it is time consuming.

Reply:
ok, done. I kept it somewhere else, but can remove it if that's better.